h2/src/main/org/h2/engine/Mode.java

```diff
@@ -551,6 +556,7 @@
         mode.nextValueReturnsDifferentValues = true;
         mode.takeGeneratedSequenceValue = true;
         mode.expressionNames = ExpressionNames.POSTGRESQL_STYLE;
+        mode.allowUsingFromClauseInUpdateStatement = true;
```

Comment:
It would be nice to add a note about this feature to `h2/src/docsrc/html/features.html`